string status
float32 target
float32 current
float32 out
float32[3] pid_terms